### ISSUE TITLE
Issue with running appium-example

### ISSUE THREAD

**Issue by scophotog** [GraphWalker/graphwalker-example#18]
Having some trouble running the appium-example
`mvn graphwalker:generate-sources` is successful

Though when I run 

`mvn graphwalker:test`
I get the following error

```
[ERROR] Failed to execute goal org.graphwalker:graphwalker-maven-plugin:4.0.0-SNAPSHOT:test 
(default-cli) on project java-appium: Execution default-cli of goal org.graphwalker:graphwalker-
maven-plugin:4.0.0-SNAPSHOT:test failed: An API incompatibility was encountered while executing 
org.graphwalker:graphwalker-maven-plugin:4.0.0-SNAPSHOT:test: java.lang.NoSuchMethodError: 
com.google.common.collect.Sets$SetView.iterator()Lcom/google/common/collect/UnmodifiableIterator;
```

I'm running Maven version 3.5.0 and Java 1.8.

**Comment by KristianKarl** [comment]
Hi,

Could you test updating the pom.xml file:
```sh
git diff
diff --git a/java-appium/pom.xml b/java-appium/pom.xml
index 0b53b52..4a99c8a 100644
--- a/java-appium/pom.xml
+++ b/java-appium/pom.xml
@@ -44,7 +44,7 @@
     <dependency>
       <groupId>io.appium</groupId>
       <artifactId>java-client</artifactId>
-      <version>3.4.1</version>
+      <version>5.0.0-BETA8</version>
     </dependency>
   </dependencies>
```

and re-run it?

**Comment by scophotog** [comment]
That fixed it. Thank you!

**Comment by KristianKarl** [comment]
Fixed by f257cb7

**Comment by KristianKarl** [comment]
Thanks for your help @scophotog
